pages/index.tsx

```diff
@@ -7,26 +7,27 @@ import Stories from '@/modules/Stories';
 import { Category, Newsroom } from '@prezly/sdk/dist/types';
 import { PageSeo } from '@/components/seo';
 import getAssetsUrl from '@/utils/prezly/getAssetsUrl';
+import { NewsroomContext } from '@/utils/prezly/context';
 
 type Props = {
     stories: Story[];
-    categories?: Array<Category>;
+    categories: Array<Category>;
```

Comment:
Let's do arrays typing consistent. We should establish which we want to use. Probably discourse topic? 🙂 
I just mean that it's a bit strange that one line above we have `T[]` and in next line it is `Array<T>`